core/ui/src/main/kotlin/com/nexters/bandalart/android/core/ui/component/BottomSheetText.kt

```diff
@@ -0,0 +1,79 @@
+package com.nexters.bandalart.android.core.ui.component
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.nexters.bandalart.android.core.ui.theme.Gray400
+import com.nexters.bandalart.android.core.ui.theme.Gray600
+import com.nexters.bandalart.android.core.ui.theme.Gray900
+import com.nexters.bandalart.android.core.ui.theme.pretendard
+
+@Composable
+fun BottomSheetTitleText(
+  modifier: Modifier = Modifier,
+  isMainCell: Boolean,
+  isSubCell: Boolean,
+) {
+  Text(
+    text = if (isMainCell) "메인 목표 수정" else if (isSubCell) "서브 목표 수정" else "태스크 수정",
+    modifier = modifier
+      .fillMaxWidth()
+      .padding(top = 20.dp),
+    textAlign = TextAlign.Center,
+    color = Gray900,
+    fontSize = 16.sp,
```

Comment:
함수 내에서 글씨 크기를 고정시키도록 해서 함수를 호출할때 16.sp 이렇게만 받도록 하는게 휴먼에러를 없앨 수 있을 거같아
```suggestion
fontSize = 16.sp.nonScaleSp
```